src/vorta/__main__.py

```diff
@@ -45,6 +49,7 @@ def exception_handler(type, value, tb):
 
     want_version = getattr(args, 'version', False)
     want_background = getattr(args, 'daemonize', False)
+    want_development = getattr(args, 'development')
```

Comment:
```suggestion
    want_development = getattr(args, 'development', False)
```

I think an exception will be raised if the attribute doesn't exist and no default value is supplied.